bundle/config/workspace.go

```diff
@@ -21,8 +21,14 @@ type Workspace struct {
 	//
 
 	// Generic attributes.
-	Host    string `json:"host,omitempty"`
-	Profile string `json:"profile,omitempty"`
+	Host               string `json:"host,omitempty"`
+	Profile            string `json:"profile,omitempty"`
+	AuthType           string `json:"auth_type,omitempty"`
+	ConfigFile         string `json:"config_file,omitempty"`
```

Comment:
Why do we need this one?

Reply:
This is to configure the location of `~./databrickscfg`. Not strictly needed for the VS Code integration but also doesn't hurt.

Happy to take it out.

Reply:
Among the other settings the config file doesn't seem portable enough. It can be different on different machines. All the other fields are externalizable / not pinned to a machine (I think, at least that should be the case).

Reply:
Would prefer not having it.

Reply:
Makes sense. I took it out.